packages/framework/src/Support/Includes.php

```diff
@@ -84,10 +85,10 @@ public static function markdown(string $filename, ?string $default = null): ?str
         $path = static::normalizePath($filename, '.md');
 
         if (! Filesystem::exists($path)) {
-            return $default === null ? null : Markdown::render($default);
+            return $default === null ? null : Markdown::render($default, HydePage::class);
         }
 
-        return Markdown::render(Filesystem::get($path));
+        return Markdown::render(Filesystem::get($path), HydePage::class);
```

Comment:
Considered if we should try to inherit the rendered page's class here, but don't think it would offer value. I'm also thinking that we may actually want to have some different logic here (maybe passing the base Markdown document class) since we probably do not want Torchlight attributions injected into partials, as they could get repeated a lot.

Reply:
Resolved in f15bea8eb839e0121b166a9ed30c6b89811d0104 and 83c613649bcb2199a452251254e2d17479ed81ed